public/css/forms.css

```
@@ -0,0 +1,129 @@
+* {
+    padding: 0;
+    margin: 0;
+}
+
+.navbar {
+    display: flex;
+    justify-content:space-between;
+    font-family: 'Exo 2', sans-serif;
+    align-items: center;
+    padding: 1.25rem 4rem;
+    background: #4B357A;
+}
+
+.logo a {
+    color: #ffffff;
+    font-family: 'Exo 2', sans-serif;
+    font-weight: bold;
+    font-size: 36px;
+}
+
+.profile {
+    font-size: 24px;
+    color: white;
+    height: 60px;
+}
+
+.profile_pic {
+    height: 60px;
+    width: 60px;
+    object-fit: contain;
+}
+
+.profile_name {
+    position: relative;
+    bottom: 15px;
+    left: 20px
+}
+
+.form-container {
+    display: flex;
+    justify-content: center;
+    font-family: 'Exo 2', sans-serif;
+    position: absolute;
+    top: 50%;
+    left: 50%;
+    margin-right: -50%;
+    transform: translate(-50%, -50%);
```

Это не очень хороший способ центровать див,  скорее это выстрел себе в ногу. В данном случае скорее всего лучше использовать
```suggestion
    margin: 0 auto;
```
А с отступом сверху уже придумать что-то другое, учитывайте что бывают очень разные размеры экранов